packages/devtools-client-adapters/types.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+/**
+ * Breakpoint
+ *
+ * @memberof actions/types
+ * @static
+ */
+export type Breakpoint = {
+    id: string,
+    location: Location,
+    loading: boolean,
+    disabled: boolean,
+    text: string,
+    condition: ?string,
+};
+
+/**
+ * BreakpointResult
+ *
+ * @memberof actions/types
+ * @static
+ */
+export type BreakpointResult = {
+    ids: string,
```

Comment:
NOTE: we tried to make **flow** fail by adding this error, but when we run flow we got `no errors`